### ISSUE TITLE
Add notes to scanner recommendations

### ISSUE THREAD

**Issue by RotasOpera** [paperless-ngx/paperless-ngx#1236]
### Description

As a owner of a Canon Maxify MB5450 I can not recommend this device and similar ones in good conscious for paperless usage. 
The device has multiple outstanding issues like only supporting smb1 and NTLMv1 auth. 
Also since the consume directory is always empty I have a bunch of documents named SCN_0001 and no way to change this behaviour.

Either remove the device recommendation or add notes to the table and hint that all Canon Scanners from the Series Maxify MB2xxx and MB5xxx are not able to scan to SMB2/3 shares or without NTLMv1 authentication. 

You would not want users buying this product based on this documentations recommendation while these issues with the device are still outstanding.

### Steps to reproduce

N/A

### Paperless-ngx version

Documentation

### Host OS

N/A

### Installation method

Other (please describe above)

**Comment by qcasey** [comment]
Hello, thanks for reporting. I believe eingemaischt is the one who added that to the table via #984. @eingemaischt do you mind confirming if you notice this bug on the 5350 in particular?

Either way, I think it's a good idea to mention the potential for smb1 and NTLMv1 auth problems in the docs.

> You would not want users buying this product based on this documentations recommendation while these issues with the device are still outstanding.

Generally agreed.

**Comment by eingemaischt** [comment]
Absolutely, this is a real problem. The alternative is to use the "scan to email"-function, but, like many other semi-professional devices, it uses a cloud-service from canon to send the email. 

The advantage of this print/scan-family is that they allow duplex scanning without turning through the adf.

Maybe add a notice to the chart?

**Comment by RotasOpera** [comment]
I get the advantage of these scanners, hence why I own one myself. 
Maybe instead of endlessly adding footnotes to the table, adding another column would help keep things a bit cleaner and would make it easier to list issues and advantages of the device.

**Comment by shamoon** [comment]
We've moved the scanner / software docs to a wiki page here that users can freely edit, thank you: https://github.com/paperless-ngx/paperless-ngx/wiki/Scanner-&-Software-Recommendations

**Comment by donulfo** [comment]
I realized the same issues with the Canon Multify MB5450. 
issue #1 - SMB v1
issue #2 which is also mentioned in the wiki regarding the process that the scanner creates the file , writes each a page into it, closes, opens... etc...  was also not working out of the box with the inotify feature. Therefore i thought why not scanning to a "proxy-folder" which is monitored by a python script. 
Each time the newly created pdf file is changed , the timer is set back. 
Nice to know here, the code was fully created with chat-gpt-3.

Just want to share the code here - maybe it helps here and there. :)


```
import time
import shutil
from watchdog.observers import Observer
from watchdog.events import FileSystemEventHandler

class NewFileHandler(FileSystemEventHandler):
    def __init__(self):
        super().__init__()
        self.files_to_move = {}

    def on_created(self, event):
        if event.is_directory:
            print()
            #print("directory created:{}".format(event.src_path))
        else:
            if event.src_path.endswith('.pdf'):
                print("file created:{}".format(event.src_path))
                self.files_to_move[event.src_path] = time.time()

    def on_modified(self, event):
        if event.is_directory:
            print()
            #print("directory modified:{}".format(event.src_path))
        else:
            if event.src_path.endswith('.pdf'):
                print("file modified:{}".format(event.src_path))
                self.files_to_move[event.src_path] = time.time()

    def check_timeout(self):
        current_time = time.time()
        for file_path, timestamp in list(self.files_to_move.items()):
            if current_time - timestamp > 45:
                # move the file
                print(f"Moving {file_path} to /PaperLess_Eingang/")
                shutil.move(file_path, '/PATHTOyourfinal-PaperLess_Consume/')
                self.files_to_move.pop(file_path)

if __name__ == "__main__":
    print("Welcome to the File Mover program for canon multify printers using paperless!")
    print("it moves the file only when all pages are added to the scan")
    print()
    print("This program will monitor the folder '/path/to/FOLDER/Canon_PaperlessEingang' for new .pdf files and move them to the '/PaperLess_Eingang' folder.")
    print()

    print("Press Ctrl+C to exit the program.")

    observer = Observer()
    event_handler = NewFileHandler()
    observer.schedule(event_handler, path='/path/toFOLDER/Canon_PaperlessEingang', recursive=True)
    observer.start()

    try:
        while True:
            event_handler.check_timeout()
            time.sleep(1)
    except KeyboardInterrupt:
        observer.stop()
    observer.join()
```

**Comment by eingemaischt** [comment]
With a recent version, you no longer need a helper code. Please have a look at the configuration parameters, you can configure the consumer to wait a specific time before consuming a file....

**Comment by goldjunge91** [comment]
Hi I’m new to paperless,
And have a Maxify MB5450 exist any guide how I set the printer correct with my paperless server ?

**Comment by shamoon** [comment]
Going to lock this as it’s not relevant any more. @goldjunge91 feel free to open a discussion
